metabase/metabase
# Elevate selected filter field values

Product doc https://www.notion.so/metabase/Show-the-selected-filter-values-on-top-of-multi-select-list-6e3f2e1880dc40bc8c50170236ca0aaa

Elevate selected field values on initial render when rendering them as a list of checkboxes.

How to verify:
- New -> Question -> Products
- Add a filter Category -> Gizmo
- Click on the added filter
- See that the Gizmo is at the top of the list
- Try checking and unchecking options. Gizmo stays at the same position, other options also don't jump.

<img width="463" alt="Screenshot 2024-03-13 at 13 37 01" src="https://github.com/metabase/metabase/assets/8542534/404d6306-0897-4e46-ac06-394e36dbdc68">
<img width="439" alt="Screenshot 2024-03-13 at 13 37 07" src="https://github.com/metabase/metabase/assets/8542534/dbcc4ecc-cbdf-4e53-ba9b-51049fc4e176">
